### PR TITLE
slow down dummy test job

### PR DESCRIPTION
**Pull request by zzpmiracle**

### CODE REVIEW

File: .github/workflows/cronjobs.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: Cronjobs
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '* * * * *' # every minutes
+    - cron:  '0 * * * *' # every hours
```
Comment (Yancey1989) [inline .github/workflows/cronjobs.yml:8]: What is this dummy job for?

Comment (zzpmiracle) [reply .github/workflows/cronjobs.yml:8]: Test for cronjob now

Comment (Yancey1989) [reply .github/workflows/cronjobs.yml:8]: maybe don't merge into the master branch if it's only a test job, better move to a personal repo or a forked repo.

Comment (fortianyou) [reply .github/workflows/cronjobs.yml:8]: I had discussed this with @Yancey1989 offline. I agree that dummy cronjob is useless for BladeDISC. But it helps us to have a quick test on the cronjob workflow for agile development purposes. We can delete the dummy cronjob after the benchmark cronjob is set up.

Comment (zzpmiracle) [reply .github/workflows/cronjobs.yml:8]: OK, I run benckmark in other branch for now.

**Pull request closed without merging**